docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,11 +6,14 @@
 
 ## Checklist:
 
-_these will eventually be enforced by CircleCI; we need to set that up at the USDOJ org level_
+### Author
 
-+ [ ] If front end or functionality change, run locally and check http://0.0.0.0:8000/report/.
-+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests)
++ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
```

Comment:
Adding this line because meeting and getting clarity on Acceptance Criteria is 🔑 to making sure our work flies through the review process smoothly.